src/views/Gallery.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div id="gallery-content">
+        <div id="header-mobile">
+            <div>
+                <h2>Specimen gallery</h2>
+                <h3>Self similarity telescope</h3>
+            </div>
+            <div type="button" id="change-button">
+                <span
+                    alt="Swap icon"
+                    id="change-icon"
+                    class="material-icons-sharp"
+                    >swap_horiz</span
+                >
+                <p id="change-text">Change Visualizer</p>
+            </div>
+        </div>
+
+        <h1 id="header">Specimen gallery</h1>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
```

Comment:
Ultimately these `<h2>` headings will be auto-generated from the list of specimens for the gallery, correct? Just a question, no action item.

Reply:
Yes, that's correct. Every visualizer that the user has either saved or is featured will get its own programmatically generated title.